src/ood-preview/lib/navigation.ml

```diff
@@ -35,8 +35,8 @@ let t =
         ; { label = "Books"; url = "/books"; icon = ""; text = "" }
         ; { label = "Events"; url = "/events"; icon = ""; text = "" }
         ; { label = "Videos"; url = "/videos"; icon = ""; text = "" }
+        ; { label = "watch"; url = "/watch"; icon = ""; text = "" }
         ; { label = "Tutorials"; url = "/tutorials"; icon = ""; text = "" }
-        ; { label = "Workshops"; url = "/workshops"; icon = ""; text = "" }
```

Comment:
Same as the comment above, this shouldn't have been removed during the conflict resolution, you needed to merge both changes.